airflow/config_templates/airflow_local_settings.py

```diff
@@ -72,8 +76,13 @@
     'disable_existing_loggers': False,
     'formatters': {
         'airflow': {
+            'format': COLORED_LOG_FORMAT if COLORED_LOG else LOG_FORMAT,
+            'class': 'airflow.utils.log.colored_log.CustomTTYColoredFormatter' if COLORED_LOG else 'logging.Formatter'
```

Comment:
This formatter is used by all the remote log storage formats (S3, GCP etc) , so lets leave "airflow" as plain text and create an "airflow_coloured" and use that where needed instead.